api/src/Entity/SingleText.php

```diff
@@ -3,11 +3,10 @@
 namespace App\Entity;
```

Comment:
Die restlichen ContentType Entities hast du nicht kopiert. Sollten aber sowieso in einen eigenen Namespace kommen.